common/djangoapps/third_party_auth/config/waffle.py

```diff
@@ -0,0 +1,25 @@
+"""
+Waffle flags and switches for third party auth .
+"""
+
+
+from edx_toggles.toggles import LegacyWaffleSwitch, LegacyWaffleSwitchNamespace
+
+_WAFFLE_NAMESPACE = u'third_party_auth'
+_WAFFLE_SWITCH_NAMESPACE = LegacyWaffleSwitchNamespace(name=_WAFFLE_NAMESPACE, log_prefix=u'ThirdPartyAuth: ')
+
+# .. toggle_name: third_party_auth.enable_multiple_sso_accounts_association_to_saml_user
+# .. toggle_implementation: WaffleSwitch
+# .. toggle_default: False
+# .. toggle_description: If enabled than learner should not be prompted for their edX password arriving via SAML
+# and already linked to the enterprise customer linked to the same IdP."
+# .. toggle_use_cases: temporary
+# .. toggle_creation_date: 2021-01-29
+# .. toggle_target_removal_date: 2021-04-31
+# .. toggle_warnings: None.
```

Comment:
Thank you for documenting this.
FYI: For future reference, `toggle_warnings` is optional so this should be dropped and LegacyXxx classes are deprecated, so we should use the newer WaffleSwitch class.  It sounds like this was taken care of already in another PR.
Thanks again.